### PR TITLE
Add rook operator install command in ocs-operator

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Add `make install-rook` command to ocs-operator e2e test

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
cc @iamniting @malayparida2000

File: ci-operator/config/red-hat-storage/ocs-operator/red-hat-storage-ocs-operator-main.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ tests:
     - as: e2e-test
       cli: latest
       commands: |
-        export BUNDLE_FULL_IMAGE_NAME="$OO_BUNDLE"; make install-ocs; make install-noobaa
+        export BUNDLE_FULL_IMAGE_NAME="$OO_BUNDLE"; make install-ocs; make install-noobaa; make install-rook
```
Comment (iamniting) [inline ci-operator/config/red-hat-storage/ocs-operator/red-hat-storage-ocs-operator-main.yaml:50]: Lets keep it in b/w

**Review by iamniting — CHANGES_REQUESTED** [review]

File: ci-operator/config/red-hat-storage/ocs-operator/red-hat-storage-ocs-operator-main.yaml (hunk 1 above)
Comment (Nikhil-Ladha) [reply ci-operator/config/red-hat-storage/ocs-operator/red-hat-storage-ocs-operator-main.yaml]: Done

**Review by iamniting — APPROVED** [review]

**Pull request merged**